express-notes-api/index.js

```diff
@@ -0,0 +1,96 @@
+const fs = require('fs');
+const express = require('express');
+const app = express();
+const jsonData = require('./data.json');
+
+app.get('/api/notes', (req, res) => {
+  const newArray = [];
+  if (jsonData.notes === null) {
+    return newArray;
+  } else {
```

Comment:
You can remove this logic. Unless there is something wrong with the program, `jsonData.notes` will always be an object. Also, `return` does not respond to the client, it just exits the `.get()` callback.